src/core/cookies/cookies.ts

```diff
@@ -0,0 +1,7 @@
+import Cookies, { CookieAttributes } from 'js-cookie'
```

Comment:
This file is not needed - just creating additional boilerplate. User directly js-cookie in Provider